src/content/en/2020/pwa.md

```diff
@@ -4,21 +4,437 @@ part_number: II
+
+The proprietary properties we encountered frequently were gcm_sender_id Google Cloud Messaging (GCM) service we found other interesting attributes like: browser_action, DO_NOT_CHANGE_GCM_SENDER_ID [which was an attribute that substituted a comment as JSON doesn’t provide comments], scope, public path, cacheDigest.
+
+On both platforms, however, there's a long tail of properties that are not interpreted by browsers yet contain potentially useful metadata.
+
+We also found a non-trivial amount of mistyped properties; our favorite ones being variation of theme-color, Theme_color, theme-color, Theme_color and oriendation.
+
+In order for a PWA to be fruitful it needs to have a Manifest and a Service Worker. It is interesting to note that Manifests are used a lot more than SW. This is due, in large part, to the fact that CMS like wordpress, drupal and Joomla have Manifests by default.
+
+{{ figure_markup(
+  image="pwa-manifest-properties-on-service-worker-pages.png",
+  caption="Manifest properties on service worker pages.",
+  description="Bar chart showing the percentage of various manifest properties used on service worker pages by desktop and mobile. The first seven properties (`name`, `display`, `icons`, `short_name`, `start_url`, `theme_color`, and `background_color`) are used by 93 to 98% of both desktop and mobile. After this there is a sharp drop off for the remaining properties with `gcm_sender_id` used by 21.66% of desktop sites and 28.98% of mobile sites, `scope` used by 29.32% of desktop and 28.77% of mobile, `description` used by 23.32% of desktop and 18.84% mobile, `orientation` by 19.45% of desktop and 17.65% of mobile, and finally `lang` used by 12.31% of desktop and 11.01% of mobile.",
+  chart_url="https://docs.google.com/spreadsheets/d/e/2PACX-1vRRpTSA4fsHwUap-ByQ08j95uo7Zm1kY6lTSvA-DZT54g2QZ0guV7db3QyQwQgMPzsKsJ43gbzqfJst/pubchart?oid=257125618&format=interactive",
+  sheets_gid="887021927",
+  sql_file="top_manifest_properties_sw.sql"
+  )
+}}
+
+### Top Manifest display values
+
+
+{{ figure_markup(
+  image="pwa-most-used-display-values-for-service-worker-pages.png",
+  alt="Most used display values for service worker pages.",
+  caption="Most used `display` values for service worker pages.",
+  description="Bar chart showing the percenatge of pages using the 5 most common `display` values, dominated by `standalone` which is used by 86.73% of desktop and 89.28% of mobile pages, `minimal-ui` is used by 6.30% of desktop and 5.00% of mobile, `fullscreen` by 4.80% of desktop and 4.11% of mobile, 1.15% of desktop pages and 0.88% of mobile pages do not set a `display` value, and finally 1.00% of desktop and 0.72% of mobile set this to `browser`.",
+  chart_url="https://docs.google.com/spreadsheets/d/e/2PACX-1vRRpTSA4fsHwUap-ByQ08j95uo7Zm1kY6lTSvA-DZT54g2QZ0guV7db3QyQwQgMPzsKsJ43gbzqfJst/pubchart?oid=872942071&format=interactive",
+  sheets_gid="1208193001",
+  sql_file="top_manifest_display_values_sw.sql"
+  )
+}}
```

Comment:
We shouldn't just list the chart in it's own section. Have we nothing to say about this chart? Can we summarise the key points for the reader?